apps/angular-console/src/app/app.module.ts

```diff
@@ -88,7 +89,12 @@ export function initApollo(
         {
           path: '',
           children: workspaceRoutes,
-          canActivateChild: [CancelCommandGuard]
+          canActivateChild: [IsNodeJsInstalledGuard, CancelCommandGuard]
+        },
+        {
+          path: 'install-nodejs',
+          loadChildren:
+            '@angular-console/feature-install-node-js#FeatureInstallNodeJsModule'
```

Comment:
This is funky looking URL!!

Reply:
Actually, it is pretty good

Reply:
:p, don't blame me, blame the author of @angular/router

Reply:
:) I was just testing out hte new vscode github integration

https://code.visualstudio.com/blogs/2018/09/10/introducing-github-pullrequests

Turns out you can add comments, but you cannot delete them :)